src/mongocxx/test/versioned_api.cpp

```diff
@@ -26,8 +27,23 @@ using namespace mongocxx;
 // We'll format many of these examples by hand
 // clang-format off
 
+static bool is_server_v5_or_newer() {
```

Comment:
Nice, you can alternatively use the `test_util::get_max_wire_version` to determine if the wire version is greater than 13. The wire version is often used in the driver to perform server support checks.

https://github.com/mongodb/specifications/blob/master/source/wireversion-featurelist.rst is a list of wire versions mapping to server versions.